source/compliance.txt

```diff
@@ -6,48 +6,189 @@ Compliance
+Use the following {+service+} features to address compliance requirements:
 
-Examples
---------
+- :ref:`arch-center-compliance-certs`
+- :ref:`arch-center-compliance-encryption`
+- :ref:`arch-center-compliance-data-sovereignty`
+- :ref:`arch-center-compliance-backup-distribution`
+- :ref:`arch-center-compliance-backup-compliance-policy`
 
-The following examples <perform this action> using |service|
-:ref:`tools for automation <arch-center-automation>`.
+.. _arch-center-compliance-certs:
 
-These examples also apply other recommended configurations, including:
+Certifications and Compliance with Regulations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. tabs::
+To help you address security and privacy requirements, MongoDB {+service+}
+aims to deliver the highest levels of standards conformance and regulatory
+compliance.
 
-   .. tab:: Dev and Test Environments
-      :tabid: devtest
+{+service+} data platform undergoes rigorous independent audits to verify
+its security, privacy, and organizational controls. These audits help ensure
+that the platform  meets compliance and regulatory requirements, including
+those specific to highly-regulated industries.
 
-      .. include:: /includes/shared-settings-clusters-devtest.rst
+{+service+} has ISO/IEC 27001 and 9001, PCI DSS, HIPAA, GDPR, FedRAMP, and
+several other certifications. The full list of certifications is included
+in this section.
 
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
+These certifications serve as a foundation for meeting
+industry-specific regulatory requirements. Use these certifications
+to maintain security controls and ensure that your data practices align
+with necessary compliance demands.
 
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
+{+service+} maintains the following certifications:
 
-.. tabs::
+- ISO/IEC 27001:2022: Global standard for information security management systems.
+- ISO/IEC 27017:2015: Global standard for cloud-specific security controls.
+- ISO/IEC 27018:2019: Global standard for protecting sensitive data (PII) on the cloud.
+- ISO 9001:2015: Globally recognized standard for Quality Management.
+- PCI DSS: Requirements for processing and accessing credit card data.
+- HIPAA: U.S. privacy regulation safeguarding health information.
+- GDPR: General Data Protection Regulation in the EU and EEA.
+- IRAP: Australian government cybersecurity assessment framework.
+- TX-RAMP: Texas Risk and Authorization Management Program.
+- TISAX: Trusted Information Security Assessment Exchange for automotive industries.
+- HDS: French standard for hosting health data (Hébergeur de Données de Santé).
 
-   .. tab:: CLI
-      :tabid: cli
+To learn more, visit the |atlas-trust|, a comprehensive compliance and
+trust program for MongoDB {+service+}.
+
+.. _arch-center-compliance-encryption:
+
+Encryption
+~~~~~~~~~~
 
-      Content here
+Implement encryption to ensure data security and compliance across all
```

Comment:
Reframe this as "you can" because this page isn't making recommendations (compliance recs will be in a future version)
```suggestion
You can implement encryption to ensure data security and compliance across all
```